kimchi/cairo/src/runner/word.rs

```diff
@@ -0,0 +1,314 @@
+/*
+    The Cairo Language works natively for field elements in the finite field with
+    modulus 0x800000000000011000000000000000000000000000000000000000000000001
+    This is the hexadecimal value for 2 ^ 251 + 17 * 2 ^ 192 + 1
+    Our Pallas curves have 256 bits, so Cairo native instructions will fit.
+*/
+
+// TODO(querolita):
+// - Enlarge native word type to u256
+
+// Number of Cairo flags
+pub const NUM_FLAGS: usize = 16;
+
+/// A Cairo instruction for simulation
+#[derive(Clone, Copy)]
+pub struct CairoWord {
+    /// 64bit word
```

Comment:
Oh thanks, better with the dash yes. 
So, you're indeed right. At the beginning I was coding separate structures and functions depending on whether they were related to the runner (pre-computation step) or the actual proof generation, as the runner was working for native rust types instead of fields elements, unlike turshi, that should work for fields. Now that the code looks quite stable in terms of functionality, and the fact that I realised Cairo will work for fields even at the runner level, I will be "merging" or "renaming" some data structures that were somehow duplicates of each other: e.g. CairoBytecode vs CairoMemory, CairoWord vs CairoInstruction. Now, for the former I do think it will become CairoMemory. But the latter is a bit more tricky. Meaning, not all Cairo words (seen as cells in memory) actually correspond to Cairo instructions (seen as 64-bit data with a certain structure). But this is not very relevant, what's important here is that they will be merged to work directly over field elements.